### ISSUE TITLE
Rounded button?

### ISSUE THREAD

**Issue by smunshimi** [CRED-CLUB/neopop-flutter#15]
Hello,

Was experimenting with this library and would like to know if there is a way to make the buttons rounded with all the effects?

That would make a nice addition.

Thanks.

**Comment by RAFI-006** [comment]
yeah we are also looking for the same feature it would be great if Cred team introduced above feature asap

**Comment by yelkamel** [comment]
That will be very interesting indeed

**Comment by opika06** [comment]
i want round button too
